wcivf/apps/people/templates/people/includes/_person_intro_card.html

```diff
@@ -23,8 +23,12 @@ <h5>{{ object.name }} is {% if object.current_or_future_candidacies.0.party.is_i
                         {% if candidacy.elected %}
                             (elected)
                         {% endif %}
+                        {% if candidacy.previous_party_affiliations.exists and candidacy.is_welsh_run %}
+                            {% include "elections/includes/_previous_party_affiliations.html" with person_post=candidacy %}
+                        {% endif %}
```

Comment:
This isn't working (it was at some point) and I can't figure out why 😕

Reply:
This does work, but will only display for a candidate that has more than 1 current candidacies (see the conditional on line 11).
If temp hardcode that conditional to `if True` then I see:
![Screenshot 2022-04-07 at 16 40 21](https://user-images.githubusercontent.com/15347726/162238623-594669eb-5db9-4caa-9aba-33ed07e070ae.png)

Reply:
Having said that, I am unsure if we need to display it in the intro, or if just on the ballot view is enough? Maybe something for @pmk01 to think about?

Reply:
Discussed this PR with @pmk01: Since the language regarding previous parties on the Welsh SOPNs is a bit vague (party affiliations aren't always previous), we discussed the following options: 

1. A explanatory sentence on the person intro card which appears on the person detail view, something like: 

> PERSON NAME has declared their membership with the following political parties in the last 12 months: Party #1, Party #2

2. Add it to the previous elections table to make it less prominent (with no mention in the person list or person detail)

@symroe @michaeljcollinsuk thoughts?

Reply:
Option 2 is probably going to be easier to implement, just because of how we build the intro. Although I dont think it would be too difficult to add to the intro as well, so happy with what people think is better.